Dockerfile

```diff
@@ -177,28 +177,6 @@ FROM build as yarn
 ARG TARGETPLATFORM
 
 # Copy Node package configuration files into working directory
```

Comment:
this comment is now a partial truth, since it also copies in Gemfiles and other stuff

Reply:
You're right it doesn't need the Gemfile in there at that part.

Reply:
You're right it doesn't need the Gemfile there.